INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /new_cargo/ HTTP/1.1" 200 7692
ERROR ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~Errors happened~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Traceback (most recent call last):
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1768, in add_fields
    join_info = self.setup_joins(name.split(LOOKUP_SEP), opts, alias, allow_many=allow_m2m)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1504, in setup_joins
    names[:pivot], opts, allow_many, fail_on_missing=True,
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1420, in names_to_path
    "Choices are: %s" % (name, ", ".join(available)))
django.core.exceptions.FieldError: Cannot resolve keyword 'cell_id' into field. Choices are: cargo, elevation, height, id, length, position, row, storage, storage_id, width

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/shared_logic/status_logger/status_logger.py", line 16, in wrapper
    res = func(*args, **kwargs)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/views.py", line 28, in new_cargo
    success = save_cargo(data)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/cargos_saving.py", line 20, in save_cargo
    print(f"{new_cell_vals.values()} !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!")
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/query.py", line 250, in __repr__
    data = list(self[:REPR_OUTPUT_SIZE + 1])
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/query.py", line 274, in __iter__
    self._fetch_all()
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/query.py", line 1242, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/query.py", line 113, in __iter__
    for row in compiler.results_iter(chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size):
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 1052, in results_iter
    results = self.execute_sql(MULTI, chunked_fetch=chunked_fetch, chunk_size=chunk_size)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 1087, in execute_sql
    sql, params = self.as_sql()
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 483, in as_sql
    result, params = self.get_combinator_sql(combinator, self.query.combinator_all)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 432, in get_combinator_sql
    *self.query.annotation_select,
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/query.py", line 2020, in set_values
    self.add_fields(field_names, True)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1791, in add_fields
    "Choices are: %s" % (name, ", ".join(names)))
django.core.exceptions.FieldError: Cannot resolve keyword 'cell_id' into field. Choices are: cargo, elevation, height, id, length, position, row, storage, storage_id, width
INFO "POST /new_cargo/ HTTP/1.1" 200 3938
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2295
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2295
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2295
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2295
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2295
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO /home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/cargos_saving.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO "GET /new_cargo/ HTTP/1.1" 200 7692
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2295
ERROR ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~Errors happened~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Traceback (most recent call last):
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1768, in add_fields
    join_info = self.setup_joins(name.split(LOOKUP_SEP), opts, alias, allow_many=allow_m2m)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1504, in setup_joins
    names[:pivot], opts, allow_many, fail_on_missing=True,
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1420, in names_to_path
    "Choices are: %s" % (name, ", ".join(available)))
django.core.exceptions.FieldError: Cannot resolve keyword 'cell' into field. Choices are: cargo, elevation, height, id, length, position, row, storage, storage_id, width

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/shared_logic/status_logger/status_logger.py", line 16, in wrapper
    res = func(*args, **kwargs)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/views.py", line 28, in new_cargo
    success = save_cargo(data)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/cargos_saving.py", line 30, in save_cargo
    storage.rows, storage.elevations, storage.positions
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/custom_logic/new_cargo_logic.py", line 12, in add_new_cargo_unformated
    print(cells)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/query.py", line 250, in __repr__
    data = list(self[:REPR_OUTPUT_SIZE + 1])
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/query.py", line 274, in __iter__
    self._fetch_all()
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/query.py", line 1242, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/query.py", line 144, in __iter__
    return compiler.results_iter(tuple_expected=True, chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 1052, in results_iter
    results = self.execute_sql(MULTI, chunked_fetch=chunked_fetch, chunk_size=chunk_size)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 1087, in execute_sql
    sql, params = self.as_sql()
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 483, in as_sql
    result, params = self.get_combinator_sql(combinator, self.query.combinator_all)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 432, in get_combinator_sql
    *self.query.annotation_select,
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/query.py", line 2020, in set_values
    self.add_fields(field_names, True)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1791, in add_fields
    "Choices are: %s" % (name, ", ".join(names)))
django.core.exceptions.FieldError: Cannot resolve keyword 'cell' into field. Choices are: cargo, elevation, height, id, length, position, row, storage, storage_id, width
INFO "POST /new_cargo/ HTTP/1.1" 200 3935
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2295
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO Watching for file changes with StatReloader
INFO "GET /new_cargo/ HTTP/1.1" 200 7692
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2295
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
ERROR ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~Errors happened~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Traceback (most recent call last):
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/shared_logic/status_logger/status_logger.py", line 16, in wrapper
    res = func(*args, **kwargs)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/views.py", line 28, in new_cargo
    success = save_cargo(data)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/cargos_saving.py", line 30, in save_cargo
    storage.rows, storage.elevations, storage.positions
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/custom_logic/new_cargo_logic.py", line 14, in add_new_cargo_unformated
    i, j, k = cell.row, cell.elevation, cell.position
AttributeError: 'tuple' object has no attribute 'row'
INFO "POST /new_cargo/ HTTP/1.1" 200 3846
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2295
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2295
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /new_cargo/ HTTP/1.1" 200 7692
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2295
INFO "GET /admin/ HTTP/1.1" 200 9352
INFO "GET /admin/cargos_main/cargo/ HTTP/1.1" 200 6706
INFO "GET /admin/jsi18n/ HTTP/1.1" 200 3223
INFO "GET /admin/cargos_main/ HTTP/1.1" 200 3376
INFO "GET /admin/cargos_main/storage/ HTTP/1.1" 200 5362
INFO "GET /admin/jsi18n/ HTTP/1.1" 200 3223
INFO /home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/custom_logic/new_cargo_logic.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO /home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/cargos_saving.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO "GET / HTTP/1.1" 200 7526
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2295
INFO "GET /new_cargo/ HTTP/1.1" 200 7692
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2295
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
ERROR ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~Errors happened~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Traceback (most recent call last):
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1768, in add_fields
    join_info = self.setup_joins(name.split(LOOKUP_SEP), opts, alias, allow_many=allow_m2m)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1504, in setup_joins
    names[:pivot], opts, allow_many, fail_on_missing=True,
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1420, in names_to_path
    "Choices are: %s" % (name, ", ".join(available)))
django.core.exceptions.FieldError: Cannot resolve keyword 'cell' into field. Choices are: cargo, elevation, height, id, length, position, row, storage, storage_id, width

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/shared_logic/status_logger/status_logger.py", line 16, in wrapper
    res = func(*args, **kwargs)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/views.py", line 28, in new_cargo
    success = save_cargo(data)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/cargos_saving.py", line 30, in save_cargo
    storage.rows, storage.elevations, storage.positions
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/custom_logic/new_cargo_logic.py", line 12, in add_new_cargo_unformated
    for cell in cells:
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/query.py", line 274, in __iter__
    self._fetch_all()
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/query.py", line 1242, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/query.py", line 144, in __iter__
    return compiler.results_iter(tuple_expected=True, chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 1052, in results_iter
    results = self.execute_sql(MULTI, chunked_fetch=chunked_fetch, chunk_size=chunk_size)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 1087, in execute_sql
    sql, params = self.as_sql()
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 483, in as_sql
    result, params = self.get_combinator_sql(combinator, self.query.combinator_all)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 432, in get_combinator_sql
    *self.query.annotation_select,
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/query.py", line 2020, in set_values
    self.add_fields(field_names, True)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1791, in add_fields
    "Choices are: %s" % (name, ", ".join(names)))
django.core.exceptions.FieldError: Cannot resolve keyword 'cell' into field. Choices are: cargo, elevation, height, id, length, position, row, storage, storage_id, width
INFO "POST /new_cargo/ HTTP/1.1" 200 3935
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2295
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO Watching for file changes with StatReloader
INFO "GET /new_cargo/ HTTP/1.1" 200 7692
ERROR ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~Errors happened~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Traceback (most recent call last):
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/shared_logic/status_logger/status_logger.py", line 16, in wrapper
    res = func(*args, **kwargs)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/views.py", line 28, in new_cargo
    success = save_cargo(data)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/cargos_saving.py", line 37, in save_cargo
    add_cargo_fields(new_row, new_el, new_pos, data)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/cargos_saving.py", line 69, in add_cargo_fields
    new_cargo_obj.date_added = parse_date_time(data.get('date_added', timezone.now()))
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/cargos_saving.py", line 56, in parse_date_time
    dta = datetime.strptime(dta, '%d.%m.%y %H:%M:%S')
TypeError: strptime() argument 1 must be str, not datetime.datetime
INFO "POST /new_cargo/ HTTP/1.1" 200 3849
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2295
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2295
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2295
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /new_cargo/ HTTP/1.1" 200 7692
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2295
INFO "GET /preview_cargos/ HTTP/1.1" 200 7929
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2295
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /preview_cargos/?page=2& HTTP/1.1" 200 8075
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2295
INFO "GET /preview_cargos/?page=3& HTTP/1.1" 200 6556
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2295
INFO "GET /preview_cargos/?page=1 HTTP/1.1" 200 7929
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2295
INFO "GET /admin/ HTTP/1.1" 200 9352
INFO "GET /admin/cargos_main/storage/ HTTP/1.1" 200 5362
INFO "GET /admin/jsi18n/ HTTP/1.1" 200 3223
INFO "GET / HTTP/1.1" 200 7526
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2295
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2295
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2295
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2295
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2295
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2295
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2295
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2295
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO /home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/cargos_saving.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /new_cargo/ HTTP/1.1" 200 7692
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2295
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "POST /new_cargo/ HTTP/1.1" 200 7692
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2295
INFO "GET /preview_cargos/ HTTP/1.1" 200 7923
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2295
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /preview_cargos/ HTTP/1.1" 200 7923
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2287
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2287
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO /home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/cargos_saving.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /preview_cargos/ HTTP/1.1" 200 7923
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2287
INFO "GET /new_cargo/ HTTP/1.1" 200 7692
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2287
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "POST /new_cargo/ HTTP/1.1" 200 7692
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2287
INFO "GET /preview_cargos/ HTTP/1.1" 200 7923
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2287
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2281
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO /home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/cargos_saving.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2281
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /preview_cargos/ HTTP/1.1" 200 7923
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2281
INFO "GET /new_cargo/ HTTP/1.1" 200 7692
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2281
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "POST /new_cargo/ HTTP/1.1" 200 7692
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2281
INFO "GET /preview_cargos/ HTTP/1.1" 200 7920
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2281
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO /home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/models.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /new_cargo/ HTTP/1.1" 200 7692
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
ERROR ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~Errors happened~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Traceback (most recent call last):
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/shared_logic/status_logger/status_logger.py", line 16, in wrapper
    res = func(*args, **kwargs)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/views.py", line 28, in new_cargo
    success = save_cargo(data)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/cargos_saving.py", line 21, in save_cargo
    print(f"{new_cell_vals}")
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/query.py", line 253, in __repr__
    return '<%s %r>' % (self.__class__.__name__, data)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/base.py", line 519, in __repr__
    return '<%s: %s>' % (self.__class__.__name__, self)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/models.py", line 38, in __str__
    return Storage.objects.get(self.storage).name +  'r: ' + str(self.row) + ' e: ' + str(self.elevation) + ' p: ' + str(self.position)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/query.py", line 399, in get
    clone = self.filter(*args, **kwargs)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/query.py", line 892, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/query.py", line 910, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1290, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1318, in _add_q
    split_subq=split_subq, simple_col=simple_col,
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1187, in build_filter
    arg, value = filter_expr
TypeError: 'Storage' object is not iterable
INFO "POST /new_cargo/ HTTP/1.1" 200 3833
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO /home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/models.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /new_cargo/ HTTP/1.1" 200 7692
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
ERROR ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~Errors happened~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Traceback (most recent call last):
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/shared_logic/status_logger/status_logger.py", line 16, in wrapper
    res = func(*args, **kwargs)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/views.py", line 28, in new_cargo
    success = save_cargo(data)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/cargos_saving.py", line 21, in save_cargo
    print(f"{new_cell_vals}")
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/query.py", line 253, in __repr__
    return '<%s %r>' % (self.__class__.__name__, data)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/base.py", line 519, in __repr__
    return '<%s: %s>' % (self.__class__.__name__, self)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/models.py", line 38, in __str__
    return Storage.objects.all().get(self.storage).name + 'r: ' + str(self.row) + ' e: ' + str(self.elevation) + ' p: ' + str(self.position)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/query.py", line 399, in get
    clone = self.filter(*args, **kwargs)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/query.py", line 892, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/query.py", line 910, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1290, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1318, in _add_q
    split_subq=split_subq, simple_col=simple_col,
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1187, in build_filter
    arg, value = filter_expr
TypeError: 'Storage' object is not iterable
INFO "POST /new_cargo/ HTTP/1.1" 200 3833
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO /home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/models.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /new_cargo/ HTTP/1.1" 200 7692
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
ERROR ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~Errors happened~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Traceback (most recent call last):
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/shared_logic/status_logger/status_logger.py", line 16, in wrapper
    res = func(*args, **kwargs)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/views.py", line 28, in new_cargo
    success = save_cargo(data)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/cargos_saving.py", line 21, in save_cargo
    print(f"{new_cell_vals}")
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/query.py", line 253, in __repr__
    return '<%s %r>' % (self.__class__.__name__, data)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/base.py", line 519, in __repr__
    return '<%s: %s>' % (self.__class__.__name__, self)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/models.py", line 38, in __str__
    return Storage.objects.all().get(storage=self.storage).name + 'r: ' + str(self.row) + ' e: ' + str(self.elevation) + ' p: ' + str(self.position)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/query.py", line 399, in get
    clone = self.filter(*args, **kwargs)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/query.py", line 892, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/query.py", line 910, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1290, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1318, in _add_q
    split_subq=split_subq, simple_col=simple_col,
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1190, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1049, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1420, in names_to_path
    "Choices are: %s" % (name, ", ".join(available)))
django.core.exceptions.FieldError: Cannot resolve keyword 'storage' into field. Choices are: cell, default_height, default_length, default_width, elevations, id, name, positions, rows
INFO "POST /new_cargo/ HTTP/1.1" 200 3949
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO /home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/models.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO "GET /new_cargo/ HTTP/1.1" 200 7692
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
ERROR ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~Errors happened~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Traceback (most recent call last):
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/shared_logic/status_logger/status_logger.py", line 16, in wrapper
    res = func(*args, **kwargs)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/views.py", line 28, in new_cargo
    success = save_cargo(data)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/cargos_saving.py", line 21, in save_cargo
    print(f"{new_cell_vals}")
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/query.py", line 253, in __repr__
    return '<%s %r>' % (self.__class__.__name__, data)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/base.py", line 519, in __repr__
    return '<%s: %s>' % (self.__class__.__name__, self)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/models.py", line 38, in __str__
    return Storage.objects.all().get(id=self.storage).name + 'r: ' + str(self.row) + ' e: ' + str(self.elevation) + ' p: ' + str(self.position)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/query.py", line 399, in get
    clone = self.filter(*args, **kwargs)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/query.py", line 892, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/query.py", line 910, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1290, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1318, in _add_q
    split_subq=split_subq, simple_col=simple_col,
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1251, in build_filter
    condition = self.build_lookup(lookups, col, value)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1116, in build_lookup
    lookup = lookup_class(lhs, rhs)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/lookups.py", line 20, in __init__
    self.rhs = self.get_prep_lookup()
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/lookups.py", line 70, in get_prep_lookup
    return self.lhs.output_field.get_prep_value(self.rhs)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/fields/__init__.py", line 968, in get_prep_value
    return int(value)
TypeError: int() argument must be a string, a bytes-like object or a number, not 'Storage'
INFO "POST /new_cargo/ HTTP/1.1" 200 3880
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO /home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/models.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO "GET /new_cargo/ HTTP/1.1" 200 7692
ERROR ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~Errors happened~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Traceback (most recent call last):
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/shared_logic/status_logger/status_logger.py", line 16, in wrapper
    res = func(*args, **kwargs)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/views.py", line 28, in new_cargo
    success = save_cargo(data)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/cargos_saving.py", line 21, in save_cargo
    print(f"{new_cell_vals}")
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/query.py", line 253, in __repr__
    return '<%s %r>' % (self.__class__.__name__, data)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/base.py", line 519, in __repr__
    return '<%s: %s>' % (self.__class__.__name__, self)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/models.py", line 38, in __str__
    return Storage.objects.all().get(self.storage).name + 'r: ' + str(self.row) + ' e: ' + str(self.elevation) + ' p: ' + str(self.position)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/query.py", line 399, in get
    clone = self.filter(*args, **kwargs)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/query.py", line 892, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/query.py", line 910, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1290, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1318, in _add_q
    split_subq=split_subq, simple_col=simple_col,
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1187, in build_filter
    arg, value = filter_expr
TypeError: 'Storage' object is not iterable
INFO "POST /new_cargo/ HTTP/1.1" 200 3833
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
INFO "GET /new_cargo/ HTTP/1.1" 200 7692
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO /home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/models.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
ERROR ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~Errors happened~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Traceback (most recent call last):
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/shared_logic/status_logger/status_logger.py", line 16, in wrapper
    res = func(*args, **kwargs)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/views.py", line 28, in new_cargo
    success = save_cargo(data)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/cargos_saving.py", line 21, in save_cargo
    print(f"{new_cell_vals}")
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/query.py", line 253, in __repr__
    return '<%s %r>' % (self.__class__.__name__, data)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/base.py", line 519, in __repr__
    return '<%s: %s>' % (self.__class__.__name__, self)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/models.py", line 38, in __str__
    return Storage.objects.all().get(id=self.storage.id).name + 'r: ' + str(self.row) + ' e: ' + str(self.elevation) + ' p: ' + str(self.position)
TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'
INFO "POST /new_cargo/ HTTP/1.1" 200 3864
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO /home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/models.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /new_cargo/ HTTP/1.1" 200 7692
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
ERROR ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~Errors happened~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Traceback (most recent call last):
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/shared_logic/status_logger/status_logger.py", line 16, in wrapper
    res = func(*args, **kwargs)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/views.py", line 28, in new_cargo
    success = save_cargo(data)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/cargos_saving.py", line 21, in save_cargo
    print(f"{new_cell_vals}")
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/query.py", line 253, in __repr__
    return '<%s %r>' % (self.__class__.__name__, data)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/base.py", line 519, in __repr__
    return '<%s: %s>' % (self.__class__.__name__, self)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/models.py", line 38, in __str__
    return Storage.objects.get(id=self.storage.id).name + 'r: ' + str(self.row) + ' e: ' + str(self.elevation) + ' p: ' + str(self.position)
TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'
INFO "POST /new_cargo/ HTTP/1.1" 200 3864
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO /home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/models.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO "GET /new_cargo/ HTTP/1.1" 200 7692
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
ERROR ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~Errors happened~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Traceback (most recent call last):
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/shared_logic/status_logger/status_logger.py", line 16, in wrapper
    res = func(*args, **kwargs)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/views.py", line 28, in new_cargo
    success = save_cargo(data)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/cargos_saving.py", line 21, in save_cargo
    print(f"{new_cell_vals}")
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/query.py", line 253, in __repr__
    return '<%s %r>' % (self.__class__.__name__, data)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/base.py", line 519, in __repr__
    return '<%s: %s>' % (self.__class__.__name__, self)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/models.py", line 38, in __str__
    return Storage.objects.get(id__exact=self.storage.id).name + 'r: ' + str(self.row) + ' e: ' + str(self.elevation) + ' p: ' + str(self.position)
TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'
INFO "POST /new_cargo/ HTTP/1.1" 200 3864
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO /home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/models.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /new_cargo/ HTTP/1.1" 200 7692
ERROR ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~Errors happened~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Traceback (most recent call last):
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/shared_logic/status_logger/status_logger.py", line 16, in wrapper
    res = func(*args, **kwargs)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/views.py", line 28, in new_cargo
    success = save_cargo(data)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/cargos_saving.py", line 21, in save_cargo
    print(f"{new_cell_vals}")
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/query.py", line 253, in __repr__
    return '<%s %r>' % (self.__class__.__name__, data)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/db/models/base.py", line 519, in __repr__
    return '<%s: %s>' % (self.__class__.__name__, self)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/models.py", line 38, in __str__
    return Storage.objects.get(pk=self.storage.pk).name + 'r: ' + str(self.row) + ' e: ' + str(self.elevation) + ' p: ' + str(self.position)
TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'
INFO "POST /new_cargo/ HTTP/1.1" 200 3864
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO /home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/models.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
INFO "GET /new_cargo/ HTTP/1.1" 200 7692
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
INFO "POST /new_cargo/ HTTP/1.1" 200 7692
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
INFO "GET /preview_cargos/ HTTP/1.1" 200 7997
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
INFO "GET /preview_cargos/?page=2& HTTP/1.1" 200 8114
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
INFO "GET /preview_cargos/?page=3& HTTP/1.1" 200 8008
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
INFO "GET /preview_cargos/?page=1 HTTP/1.1" 200 7997
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2273
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO /home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/models.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO "GET /preview_cargos/?page=1 HTTP/1.1" 200 8002
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO /home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/cargos_saving.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /new_cargo/ HTTP/1.1" 200 7692
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /preview_cargos/ HTTP/1.1" 200 8002
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /new_cargo/ HTTP/1.1" 200 7692
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
ERROR ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~Errors happened~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Traceback (most recent call last):
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/shared_logic/status_logger/status_logger.py", line 16, in wrapper
    res = func(*args, **kwargs)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/views.py", line 28, in new_cargo
    success = save_cargo(data)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/cargos_saving.py", line 31, in save_cargo
    storage.rows, storage.elevations, storage.positions
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/custom_logic/new_cargo_logic.py", line 14, in add_new_cargo_unformated
    storage[i][j][k].height, storage[i][j][k].length, storage[i][j][k].width = float(cell.height), float(cell.length), float(cell.width)
IndexError: list index out of range
INFO "POST /new_cargo/ HTTP/1.1" 200 3816
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO /home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/custom_logic/new_cargo_logic.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO "GET /new_cargo/ HTTP/1.1" 200 7692
ERROR ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~Errors happened~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Traceback (most recent call last):
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/shared_logic/status_logger/status_logger.py", line 16, in wrapper
    res = func(*args, **kwargs)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/views.py", line 28, in new_cargo
    success = save_cargo(data)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/cargos_saving.py", line 31, in save_cargo
    storage.rows, storage.elevations, storage.positions
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/custom_logic/new_cargo_logic.py", line 13, in add_new_cargo_unformated
    print(i, j, k)
UnboundLocalError: local variable 'i' referenced before assignment
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "POST /new_cargo/ HTTP/1.1" 200 3848
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO /home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/custom_logic/new_cargo_logic.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO /home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/custom_logic/new_cargo_logic.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO "GET /new_cargo/ HTTP/1.1" 200 7692
ERROR ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~Errors happened~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Traceback (most recent call last):
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/shared_logic/status_logger/status_logger.py", line 16, in wrapper
    res = func(*args, **kwargs)
  File "/home/stepan/anaconda3/envs/cargos_backend/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/views.py", line 28, in new_cargo
    success = save_cargo(data)
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/cargos_saving.py", line 31, in save_cargo
    storage.rows, storage.elevations, storage.positions
  File "/home/stepan/PycharmProjects/cargos_backend/cargos_backend/cargos_main/custom_logic/new_cargo_logic.py", line 15, in add_new_cargo_unformated
    storage[i][j][k].height, storage[i][j][k].length, storage[i][j][k].width = float(cell.height), float(cell.length), float(cell.width)
IndexError: list index out of range
INFO "POST /new_cargo/ HTTP/1.1" 200 3816
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO Watching for file changes with StatReloader
INFO "GET /users/notifyCreate/ HTTP/1.1" 200 17
INFO "GET /new_cargo/ HTTP/1.1" 200 7692
INFO "GET /inbox/notifications/api/unread_list/?max=5 HTTP/1.1" 200 2265
INFO "GET /admin/ HTTP/1.1" 200 9352
INFO "GET /admin/cargos_main/cell/ HTTP/1.1" 200 8521
INFO "GET /admin/jsi18n/ HTTP/1.1" 200 3223
INFO "GET /admin/cargos_main/ HTTP/1.1" 200 3376
INFO "GET /admin/cargos_main/storage/ HTTP/1.1" 200 5362
INFO "GET /admin/jsi18n/ HTTP/1.1" 200 3223
